app/assets/javascripts/discourse/app/components/user-profile-avatar.hbs

```diff
@@ -1,11 +1,26 @@
-<div class="user-profile-avatar">
+<PluginOutlet
+  @name="user-profile-avatar-wrapper"
+  @connectorTagName="div"
+  @outletArgs={{hash user=@user}}
+>
```

Comment:
is this outer one used/planned to be used?

Reply:
I was just making a topic on it but yeah